src/test/java/eu/europeana/metis/sandbox/service/dataset/DatasetGeneratorServiceImplTest.java

```diff
@@ -19,44 +21,79 @@
 @ExtendWith(MockitoExtension.class)
 class DatasetGeneratorServiceImplTest {
 
-  @Mock
-  private XmlRecordProcessorService xmlRecordProcessorService;
+  @Mock private XmlRecordProcessorService xmlRecordProcessorService;
 
-  @InjectMocks
-  private DatasetGeneratorServiceImpl generator;
+  @InjectMocks private DatasetGeneratorServiceImpl generator;
 
   @Test
   void generate_expectSuccess() {
+    when(xmlRecordProcessorService.getRecordId(any(byte[].class))).thenReturn("1").thenReturn("2");
 
-    when(xmlRecordProcessorService.getRecordId(any(byte[].class)))
-        .thenReturn("1")
+    Dataset dataset =
+        generator.generate(
+            "1",
```

Comment:
Could you please refrain from adding explicit new lines? 
The formatter doesn't format explicit lines atm.
(In some cases it's useful to have explicit breaks but most cases we shouldn't add them explicitly)

Reply:
Yes, I fixed my IDE with the CodeStyle from metis-framework, so this was made with GoogleStyle Java format plugin.